src/containers/FittableContainer/FittableContainer.jsx

```diff
@@ -60,24 +65,39 @@ function mapDispatchToProps (dispatch) {
     onWindowResize: () => dispatch(detectScreenSize()),
     onErrorHide: () => dispatch(hideDataError()),
     onUserRequest: () => dispatch(fetchUserData()),
+    changeCalendar: (calendar) => dispatch(changeCalendar(calendar)),
   }
 }
 
+function getSemesterData (props) {
```

Comment:
This is very misleading name for a function that actually doesn’t return anything. Functions `getX` typically returns some value.